airflow/models.py

```diff
@@ -4463,8 +4463,7 @@ class XCom(Base, LoggingMixin):
     id = Column(Integer, primary_key=True)
     key = Column(String(512))
     value = Column(LargeBinary)
-    timestamp = Column(
-        DateTime, default=timezone.utcnow, nullable=False)
+    timestamp = Column(UtcDateTime, default=timezone.utcnow, nullable=False)
```

Comment:
@bolkedebruin When you did the airflow-288/airflow-289 commits, should this have also changed?